LICENSE

```diff
@@ -19,12 +19,14 @@ Contributors include:
 
```

Comment:
Note: All contributors need to be included in LICENSE file per LLNL release policy.